### PR TITLE
Rnfixes

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vu0v50kxqymy6dnhgy8yc8uedg

[download apk](http://filestorage.bluewallet.io:1488/1590817159.apk)

File: Navigation.js [hunk 1]
```diff
@@ -214,9 +232,16 @@ const AztecoRedeemRoot = () => (
   </AztecoRedeemStack.Navigator>
 );
 
+const ScanQRCodeStack = createStackNavigator();
+const ScanQRCodeRoot = () => (
```
Comment (limpbrains) [inline Navigation.js:236]: I wonder, why do you need separate stack for ScanQRCode screen ? With just one screen in it

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dtbac215n1rfkae6ggb8u5p91g

[download apk](http://filestorage.bluewallet.io:1488/1590881418.apk)

File: Navigation.js (hunk 1 above)
Comment (marcosrdz) [reply Navigation.js:241]: Apple HIG indicates that Camera screens should be full screen rather than a card overlay.

Comment (marcosrdz) [reply Navigation.js:241]: To achieve a different transition, you gotta have a different stack

**Comment by marcosrdz** [comment]
@Overtorment  this is modal effect PR

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tnptfykrbyq47wrubbyq5qzj4w

[download apk](http://filestorage.bluewallet.io:1488/1590883043.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mvzhjw4ruayhwv276tkfgrkuw4

[download apk](http://filestorage.bluewallet.io:1488/1590896090.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ht4cu1q5vbv9wjk3q54rdd3eg8

[download apk](http://filestorage.bluewallet.io:1488/1590896571.apk)

**Comment by Overtorment** [comment]
e2e tests consistently fail:

```
    ✕ can import BIP84 mnemonic, fetch balance & transactions, then create a transaction (223260ms)
    ✕ can import zpub as watch-only and create PSBT (225658ms)
```

I looked at uploaded video & screenshot, and something is wrong on amount input when creating tx:

image

you can see that 0.0005  went into address field, and not amount field

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m50by1ndmme8nk66qz9zqgd6ug

[download apk](http://filestorage.bluewallet.io:1488/1590936696.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6h0mz7qzdrb90j3vm7pde4gyjg

[download apk](http://filestorage.bluewallet.io:1488/1590950497.apk)

**Pull request merged**